src/MySensorModel.cpp

```diff
@@ -0,0 +1,182 @@
+//
+// Copyright 2016 -- 2018 PMSF IT Consulting Pierre R. Mai
+// Copyright 2023 BMW AG
+// SPDX-License-Identifier: MPL-2.0
+// The MPL-2.0 is only an example here. You can choose any other open source license accepted by OpenMSL.

```

Comment:
You can add the hint that the templates of OpenMSL can be used internally by e.g. not giving it an OS license

Reply:
Done.